mgmt_risk/models/probability.py

```diff
@@ -9,4 +9,5 @@ class MgmtProbability(models.Model):
     _name = "mgmt.probability"
```

Comment:
Whenever a probability or a severity is created. Add combination records to `mgmt.risk.combination`.

use the `def create` and `def unlink` method.